### PR TITLE
adding member sign up

### PR DESCRIPTION
**Pull request by MandyMeindersma**
Finally finishing my pr from like months ago.

### CODE REVIEW

File: pages/become_a_member.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                        Give us $20/year to support the work we do (thank you!)
+                    </li>
+                </ul>
+            </div>
+            <p class="py-2">
+                If you're still interested in becoming a member, please complete
+                the form below and then e-transfer $20 to
+                memberships@devedmonton.com. Membership fee's are to be paid
+                annually (we will remind you 😛 ) and if you have any questions
+                about when your membership expires, these can be answered from
```
Comment (MandyMeindersma) [inline pages/become_a_member.vue:46]: Last pr we were worried about the wording of this, so I am hoping just saying refer to the bylaws will hopefully get us through :P

**Comment by MarkBennett** [comment]
Just taking a quick look at this. It's saying you need to update from the main branch.

**Comment by MandyMeindersma** [comment]
> Just taking a quick look at this. It's saying you need to update from the main branch.

I mean yeah, I am missing a commit from the all contributors but that shouldn't affect what I am doing.

**Comment by MandyMeindersma** [comment]
@MarkBennett updated now

**Comment by MarkBennett** [comment]
> @MarkBennett updated now

Awesome. I'll check it out @MandyMeindersma! :)

File: pages/become_a_member.vue [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                the bylaws.
```
Comment (MarkBennett) [inline pages/become_a_member.vue:47]:
```suggestion
               <a href="https://docs.google.com/document/d/1i6oXabir-628csa2bQacHpc17Kmhec9EfkyHij9yQGM/edit#heading=h.sns5qzyi4gru">the bylaws</a>.
```

**Review by MarkBennett — CHANGES_REQUESTED** [review]
Looks great @MandyMeindersma. I like that this is up in one place that we can send interested people to now, and that it will explain why they might not want to become a member.

One small suggestion to add a link to the bylaws, then I'll sign off.

**Comment by MandyMeindersma** [comment]
@MarkBennett all finished 👍

**Review by MarkBennett — APPROVED** [review]
LGTM! Ship it! 🥳

**Pull request merged**